nbbuild/antsrc/org/netbeans/nbbuild/ReleaseJsonProperties.java

```diff
@@ -114,9 +114,9 @@ public void execute() throws BuildException {
         } catch (ParseException | IOException ex) {
             throw new BuildException(ex);
         }
-        
+
         // remove empty api doc 
-        ri.removeIf(e -> e.apidocurl.isEmpty());
+        //ri.removeIf(e -> e.apidocurl.isEmpty());
```

Comment:
I just spent 15min looking for the build issue that @JaroslavTulach had with older VSNetBeans release to realise it's already covered by this change! :smile:

Reply:
@JaroslavTulach normally I'd be the first to agree with you on data duplication, but as also mentioned there, we're losing the canonical record of Apache IDE releases in that file, which is also important because it's used elsewhere. Let's keep conversation on that there.